client/src/components/ElementEditor/HoverBar.scss

```diff
@@ -1,65 +1,82 @@
-.element-editor__add-block-area-container {
+$transition: all ease .2s;
+
+.hover-bar {
```

Comment:
Can you make this more unique? Maybe something like `.element-editor__hover-bar`

Reply:
Presumably, this is to avoid having the CSS bleed into other part of the CMS?

Reply:
Yeah, but it also follows the existing naming convention for other parts of elemental

Reply:
Will do ... wasn't aware of this convention in elemental.

Reply:
Done.